bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java

```diff
@@ -11,6 +15,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import de.fraunhofer.abm.app.EmailConfigInterface;
```

Comment:
If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove.